src/main/java/org/carlmontrobotics/commands/Outtake.java

```diff
@@ -9,39 +12,56 @@
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.Command;
 
-public class SwitchRPMShootNEO extends Command {
+public class Outtake extends Command {
     private IntakeShooter intakeShooter;
+    private Arm arm;
+    private double currentPos;
     private double rpmAmount;
     private Timer timer = new Timer();
-   
-    
-    public SwitchRPMShootNEO(IntakeShooter intakeShooter) {
+
+
+    public Outtake(IntakeShooter intakeShooter, Arm arm) {
         this.intakeShooter = intakeShooter;
+        this.arm = arm;
         addRequirements(intakeShooter);
     }
+
     @Override
     public void initialize() {
         intakeShooter.setMaxOuttake(1);
-        timer.reset();
+        currentPos = arm.getArmPos();
+        if (currentPos >= PODIUM_ANGLE_RAD + 0.1
+                && currentPos <= PODIUM_ANGLE_RAD - 0.1) {
+            rpmAmount = PODIUM_RPM;
+        } else if (currentPos >= SPEAKER_ANGLE_RAD - 0.1) {
+            rpmAmount = SPEAKER_RPM;
+        } else {
+            rpmAmount = SUBWOOFER_RPM;
+        }
```

Comment:
shouldn't it be currentPos <= PODIUM_ANGLE_RAD + 0.1 instead of >=? Also you could just do `Math.abs(currentPos - PODIUM_ANGLE_RAD) <= 0.1` for simplicity.

Reply:
unimportant.